src/domain/entities/team.ts

```diff
@@ -5,12 +5,12 @@ export enum MemberRole {
   /**
    * Team member can read and write notes
```

Comment:
Description is not clear. Why _user can read and write_, but the variable is named just "read"?

Reply:
Oops :)